lredis/cqueues.lua

```diff
@@ -33,6 +35,22 @@ local function connect_tcp(host, port)
 	return new(socket)
 end
 
+local uri_lpeg = uri_lib.uri* lpeg.P(-1)
+local function connect(url)
+	url = url or ""
+	local m = uri_lpeg:match(url) or uri_lpeg:match("redis://"..url)
+	m.scheme = m.scheme or "redis"
+	assert(m.scheme == "redis", "Expected uri scheme to be 'redis', but got '"..m.scheme.."'")
+	local self = connect_tcp(m.host, m.port)
+	if m.userinfo then
+		self:call("AUTH", m.userinfo:match("^.*:(.*)"))
+	end
+	if m.path and m.path ~= "/" then
+		self:call("SELECT", m.path:match("^/(.*)"))
```

Comment:
should you validate that `path` is a valid db argument? (i.e. positive decimal integer with no leading zeros)